packages/odyssey-react-mui/src/TextField.tsx

```diff
@@ -111,7 +84,8 @@ export type TextFieldProps = {
    * The value of the `input` element, required for a controlled component.
    */
   value?: string;
-} & SeleniumProps;
+} & SeleniumProps &
+  FieldComponentProps;
```

Comment:
I would prefer we swap the order of these. I think it changes how these appear when hovering the component.